### ISSUE TITLE
Feature Request: Support graphql schema export by postgres schemas

### ISSUE THREAD

**Issue by Jerry-Hong** [hasura/graphql-engine#2747]
Currently graphQL schema export all schemas. Is it possible to customize the export graphql schema scope by PostgreSQL schema, like change url `/v1/graphql` (all schemas) -> `public/v1/graphql` (only public schema) or add in header?

**Comment by mnlbox** [comment]
You can untrack tables to prevent export.

**Comment by Jerry-Hong** [comment]
@mnlbox yes, but untrack need to remove all relations of the table 😢 My temporary solution is create a lot of roles and setting permissions for each schema to each role, it is also repetitive work.
